#!/bin/bash

PATCH="slontoo.patch"
for i in sys-apps/openrc/openrc-*.ebuild
	cat > ${PATCH} << EOF
--- a/${i}	2016-01-26 17:48:37.266772602 +0100
+++ b/${i}	2016-01-26 17:53:46.576752348 +0100
@@ -88,8 +88,8 @@
 		brand="Linux"
 	elif use kernel_FreeBSD ; then
 		MAKE_ARGS="${MAKE_ARGS} OS=FreeBSD"
 		brand="FreeBSD"
 	fi
-	export BRANDING="Gentoo ${brand}"
+	export BRANDING="Slontoo ${brand}"
 	use prefix && MAKE_ARGS="${MAKE_ARGS} MKPREFIX=yes PREFIX=${EPREFIX}"
 	export DEBUG=$(usev debug)
EOF
	git apply ${PATCH} || exit 1
	rm ${PATCH}
done
